tests/test_hubspot_bookmarks.py

```diff
@@ -57,6 +57,8 @@ def create_test_data(self, expected_streams):
         for stream in expected_streams - {'contacts_by_company'}:
             if stream == 'contacts': 
                 self.times=10
+            elif stream == 'contact_lists':
+                self.times=2
```

Comment:
I'm not sure why we are changing the number of times we are running this, but IIRC we need at least three so that we can add 3 on the first sync and then remove one record and update one record which wouldn't be synced on the next sync.  And the last item added is usually resynced on the next sync. So you need 3 records.  I'm curious if you meant this to be static_contact_lists since we create an extra one of those?

Reply:
yes, that is the idea. I want one of the 3 to be a static_contact_list